jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/signin.js

```diff
@@ -40,18 +40,11 @@ $(function () {
         var footerContent = '<input  type="submit" id="signin-submit-button" value="' + $labelSignin + '" class="btn btn-primary" name="commit"/>';
 
         var submitDialog = function (e) {
-            if (e.keyCode == enterCode) {
-                //if focus on username then select password field
-                if ($(e.target).is('#userName')) {
+            if((e.keyCode || e.charCode) == enterCode) {
```

Comment:
According to this http://www.w3schools.com/jsref/event_key_charcode.asp it's e.key that should be used instead of `e.keyCode` and `e.charCode`. But the latter should still be checked for backward compatibility. 

Could you please create a global function that can return the key code and it checks all the 3 options? And also leave comments that some of the options are used for backward compatibility.